README.md

```diff
@@ -24,6 +25,10 @@ Log events will be printed to the debug output:
 
 ![Debug Output](https://raw.githubusercontent.com/serilog/serilog-sinks-debug/dev/assets/Screenshot.png)
 
+The chaining of the `MinimumLevel.Debug()` call is required in order to log events of level `LogEventLevel.Debug`, as the default minimum level for a new `LoggerConfiguration` is `LogEventLevel.Information`.
```

Comment:
Perhaps instead of this, if we got a screenshot that did not include Debug output?

Or linked: 

https://github.com/serilog/serilog/wiki/Configuration-Basics#minimum-level

with a message like

    ⚠️ The default minimum log level is `Information`. To output `Debug` (`DBG` in the above screenshot), the `LoggerConfigation` must opt into the lower level, or the log messages won't enter the output pipeline.